OAuth/Response/LinkedinUserResponse.php

```diff
@@ -68,6 +69,7 @@ public function getProfilePicture()
      */
     protected function getPreferredLocaleValue($path)
     {
+        /** @var array $multiLocaleString */
```

Comment:
Shouldn't it be?
```suggestion
        /** @var array<string, string[]> $multiLocaleString */
```

Reply:
Checking https://docs.microsoft.com/en-us/linkedin/shared/references/v2/object-types#multilocalestring

its actually `array<string, array<string, string>>`, or?